### PR TITLE
Change the QUICK START guide and add 'Have a try via TiUP' section

### PR DESCRIPTION
**Pull request by lonng**
Signed-off-by: Lonng <heng@lonng.org>

### What problem does this PR solve?

Change the `README.md`, and add a `Have a try via TiUP` section into `QUICK START` part.

### Check List 

Tests 

- No code

### Release note 

- No need

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,7 +59,26 @@ Read the [Roadmap](https://pingcap.com/docs/ROADMAP).
...
+    tiup playground
+    ```
+    
+  - Run a nightly TiDB cluster (the nightly built version)
+  
+    ```shell script
+    tiup playground nightly
+    ```
+
+- Read the [Quick Start Guide](https://pingcap.com/docs/QUICKSTART), which includes deployment methods using Ansible, Docker, and Kubernetes.
```
Comment (ran-huang) [inline README.md:81]: The link directs to the v3.0 overview, not a quickstart guide. Please check and update it.

Comment (lonng) [reply README.md]: PTAL again.

File: README.md [hunk 2]
```diff
@@ -59,7 +59,26 @@ Read the [Roadmap](https://pingcap.com/docs/ROADMAP).
...
+    tiup playground
+    ```
+    
+  - Run a nightly TiDB cluster (the nightly built version)
+  
+    ```shell script
+    tiup playground nightly
+    ```
+
+- For other ways, see [TiDB documentation](https://pingcap.com/docs/).
```
Comment (ran-huang) [inline README.md:81]: How about this?
```suggestion
- For other methods, see [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/).
```

File: README.md [hunk 3]
```diff
@@ -59,7 +59,26 @@ Read the [Roadmap](https://pingcap.com/docs/ROADMAP).
...
+    tiup playground
+    ```
+    
+  - Run a nightly TiDB cluster (the nightly built version)
+  
+    ```shell script
+    tiup playground nightly
+    ```
+
+- For other methods, see [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/).
```
Comment (zz-jason) [inline README.md:81]: How about:

## Quick start

- Install [TiUP](https://tiup.io)

  ```shell script
  curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh 
  ```

- Run a nightly TiDB cluster (the nightly built version)

  ```shell script
  tiup playground nightly
  ```

See [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/) for other quick start methods.

Comment (lonng) [reply README.md]: Awesome advice, addressed.

File: README.md [hunk 4]
```diff
@@ -59,7 +59,25 @@ Read the [Roadmap](https://pingcap.com/docs/ROADMAP).
...
+    tiup playground
+    ```
+
+- Run a nightly TiDB cluster (the nightly built version)
+    
+    ```shell script
+    tiup playground nightly
+    ```
+
+See [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/) for other quick start methods.
```
Comment (bb7133) [inline README.md:80]:
```suggestion
See [TiDB Quick Start Guide](https://pingcap.com/docs/stable/quick-start-with-tidb/) for other quick start ways.
```
How about this?

Comment (lonng) [reply README.md:80]: em, see https://github.com/pingcap/tidb/pull/17188#discussion_r424844436

**Comment by zz-jason** [comment]
I'm going to close this PR since it's hasn't been updated for a long time, feel free to reopen it if you are planning to continue this PR in the future. Thanks for your contribution.

**Pull request closed without merging**